runner/operator/access/__init__.py

```diff
@@ -59,14 +59,14 @@ def get_request_id_runs(app, run_ids, request_id):
     return most_recent_runs_for_request, request_id
 
 
-def create_cwl_file_object(file_path):
+def create_cwl_file_object(file_path, url="juno://"):
```

Comment:
Should this be changed? @buehlere

Reply:
Yes I set juno:// as the default so I didn't have to change the function call in the old juno operators: https://github.com/mskcc/beagle/blob/feature/IRIS_update/runner/operator/access/v1_0_0/msi/__init__.py#L124-L125.

In the newest operators for iris, I set the url parameter: https://github.com/mskcc/beagle/blob/feature/IRIS_update/runner/operator/access/v1_IRIS/msi/__init__.py#L116-L117